Atari-Game-Player/README.md

```diff
@@ -0,0 +1,46 @@
+# Atari - Solving Games with AI 🤖
+
+The purpose of this repository is to demonstrate how ZenML enables you to build complex reinforcement learning application in fairly simple way and we had made use of DQN Agent that solves Atari Games.
+
+ZenML is an extensible, open-source MLOps framework to create production-ready machine learning pipelines. Built for data scientists, it has a simple, flexible syntax, is cloud- and tool-agnostic, and has interfaces/abstractions that are catered towards ML workflows.
+
+At its core, ZenML pipelines execute ML-specific workflows from sourcing data to splitting, preprocessing, training, all the way to the evaluation of results and even serving. There are many built-in batteries to support common ML development tasks. ZenML is not here to replace the great tools that solve these individual problems. Rather, it integrates natively with popular ML tooling and gives standard abstraction to write your workflows.
+
+Within this repo, we will use ZenML to build pipelines for developing One of the most powerful reinforcement learning algorithms which is DQN, which leans to solve Atari Games using AI.

```

Comment:
Maybe say what the acronym DQN means and link to the paper? Might make sense to even cite the paper